### ISSUE TITLE
Fresh pdb_sync fails with ValidationError 

### ISSUE THREAD

**Issue by benmaddison** [peeringdb/django-peeringdb#10]
Using `django-peeringdb (0.6.0)`
With empty DB and freshly made migrations:

```
(prngmgr-venv) benm@benm-laptop:~/venvs/prngmgr-venv/myprngmgr$ ./manage.py pdb_sync
org last update 0 11021 changed
data to be processed 11021
fac last update 0 2555 changed
data to be processed 2555
Traceback (most recent call last):
  File "./manage.py", line 10, in <module>
    execute_from_command_line(sys.argv)
  File "/home/benm/venvs/prngmgr-venv/local/lib/python2.7/site-packages/django/core/management/__init__.py", line 364, in execute_from_command_line
    utility.execute()
  File "/home/benm/venvs/prngmgr-venv/local/lib/python2.7/site-packages/django/core/management/__init__.py", line 356, in execute
    self.fetch_command(subcommand).run_from_argv(self.argv)
  File "/home/benm/venvs/prngmgr-venv/local/lib/python2.7/site-packages/django/core/management/base.py", line 283, in run_from_argv
    self.execute(*args, **cmd_options)
  File "/home/benm/venvs/prngmgr-venv/local/lib/python2.7/site-packages/django/core/management/base.py", line 330, in execute
    output = self.handle(*args, **options)
  File "/home/benm/venvs/prngmgr-venv/local/lib/python2.7/site-packages/django_peeringdb/management/commands/pdb_sync.py", line 89, in handle
    self.sync(tables, pk, limit=limit)
  File "/home/benm/venvs/prngmgr-venv/local/lib/python2.7/site-packages/django_peeringdb/management/commands/pdb_sync.py", line 96, in sync
    self.update_db(cls, self.get_objs(cls, pk=pk, **kwargs))
  File "/home/benm/venvs/prngmgr-venv/local/lib/python2.7/site-packages/django_peeringdb/management/commands/pdb_sync.py", line 175, in update_db
    self._sync(cls, row)
  File "/home/benm/venvs/prngmgr-venv/local/lib/python2.7/site-packages/django_peeringdb/management/commands/pdb_sync.py", line 144, in _sync
    sync.sync_obj(cls, row)
  File "/home/benm/venvs/prngmgr-venv/local/lib/python2.7/site-packages/django_peeringdb/sync.py", line 32, in sync_obj
    raise e
django.core.exceptions.ValidationError: {'latitude': [u'Ensure that there are no more than 9 digits in total.'], 'longitude': [u'Ensure that there are no more than 9 digits in total.']}
```

**Comment by benmaddison** [comment]
Issue is not present in `0.5.0`

**Comment by benmaddison** [comment]
Probably introduced by:

```diff
+    latitude = models.DecimalField(max_digits=9, decimal_places=6, blank=True, null=True, help_text="Latitude")
+    longitude = models.DecimalField(max_digits=9, decimal_places=6, blank=True, null=True, help_text="Longitude")
```

**Comment by vegu** [comment]
Seems to be some odd validation error with the geocoding fields added in 0.6.0, thanks for the report.

**Comment by benmaddison** [comment]
Confirmed fixed.
@vegu please would you tag a release when you have time - I need to update dependencies for another package.

**Comment by vegu** [comment]
@benmaddison - released as 0.6.1
